File: app/src/main/java/com/sagar/watchnext/network/models/movies/videos/Video.java

package com.sagar.watchnext.network.models.movies.videos;

import com.google.gson.annotations.SerializedName;

public class Video {

    @SerializedName("id")
    private String id;

    @SerializedName("iso_639_1")
    private String iso6391;

    @SerializedName("iso_3166_1")
    private String iso31661;

    @SerializedName("key")
    private String key;

    @SerializedName("name")
    private String name;

    @SerializedName("site")
    private String site;

    @SerializedName("size")
    private int size;

    @SerializedName("type")
    private String type;

    public String getId() {
        return id;
    }

    public String getIso6391() {
        return iso6391;
    }

    public String getIso31661() {
        return iso31661;
    }

    public String getKey() {
        return key;
    }

    public String getName() {
        return name;
    }

    public String getSite() {
        return site;
    }

    public int getSize() {
        return size;
    }

    public String getType() {
        return type;
    }
}

File: app/src/main/java/com/sagar/watchnext/network/models/tv/videos/Video.kt
package com.sagar.watchnext.network.models.tv.videos

import com.google.gson.annotations.SerializedName

class Video {

    @SerializedName("id")
    val id: String? = null

    @SerializedName("iso_639_1")
    val iso6391: String? = null

    @SerializedName("iso_3166_1")
    val iso31661: String? = null

    @SerializedName("key")
    val key: String? = null

    @SerializedName("name")
    val name: String? = null

    @SerializedName("site")
    val site: String? = null

    @SerializedName("size")
    val size: Int = 0

    @SerializedName("type")
    val type: String? = null
}
